mod_modular_vanilla/hooks/entity/world/player_party.nut

```diff
@@ -0,0 +1,36 @@
+::ModularVanilla.MH.hook("scripts/entity/world/player_party", function (q) {
+	// MV: Modularized
+	q.updateStrength = @() function()
+	{
+		this.m.Strength = 0.0;
+		local roster = ::World.getPlayerRoster().getAll();
+
+		if (roster.len() > ::World.Assets.getBrothersScaleMax())
+		{
+			roster.sort(this.onLevelCompare);
+		}
+
+		if (roster.len() < ::World.Assets.getBrothersScaleMin())
+		{
+			this.m.Strength += 10.0 * (::World.Assets.getBrothersScaleMin() - roster.len());
+		}
```

Comment:
```suggestion
		if (roster.len() < ::World.Assets.getBrothersScaleMin())
		{
			this.m.Strength += 10.0 * (::World.Assets.getBrothersScaleMin() - roster.len());
		}

		if (roster.len() > ::World.Assets.getBrothersScaleMax())
		{
			roster.sort(this.onLevelCompare);
		}
```

I would swap these two as the levelCompare has nothing to do with the enforcement of the min scaling

Reply:
I just copied the vanilla function and would prefer to leave it like it is unless there is an important reason to change it.